frontend/pages/admin/pacta-version/new.vue

```diff
@@ -17,7 +17,7 @@ const pactaVersion = useState<PactaVersion>(`${prefix}.pactaVersion`, () => ({
 
 const discard = () => router.push('/admin/pacta-version')
 const save = () => withLoadingAndErrorHandling(
-  () => pactaClient.createPactaVersion(pactaVersion.value).then(handleOAPIError),
+  () => pactaClient.createPactaVersion(pactaVersion.value).then(handleOAPIError).then(() => router.push('/admin/pacta-version')),
```

Comment:
Do you not want to navigate directly to the new version?

Reply:
Nope - the new version is essentially the page we were on (i.e. just contains the editor). This will bring them back to see their new version in context.